src/features/Treege/components/Forms/ReceiveValueFromAncestor.tsx

```
@@ -0,0 +1,62 @@
+
+  const handleChange = (event: SelectChangeEvent<string | undefined>) => {
+ const { value: newValue } = event.target;
+    const selectedAncestor = ancestors.find((a) => a.name === newValue);
+    const selectedUuid = selectedAncestor?.uuid;
+
+    onChange?.(selectedUuid, newValue);
+  };
+
+  return (
+    <Stack spacing={1} pb={2} pt={3}>
+  <InputLabel id={labelId}> {t("receiveValueFromParent")}</InputLabel>
```

ESLint

[error] Insert `····`

(prettier/prettier)